homeassistant/components/lamarzocco/entity.py

```diff
@@ -0,0 +1,50 @@
+"""Base class for the La Marzocco entities."""
+
+from dataclasses import dataclass
+
+from lmcloud.const import LaMarzoccoModel
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import LaMarzoccoUpdateCoordinator
+
+
+@dataclass(frozen=True, kw_only=True)
+class LaMarzoccoEntityDescription(EntityDescription):
+    """Description for all LM entities."""
+
+    supported_models: tuple[LaMarzoccoModel, ...] = (
```

Comment:
Why is this needed in the entity description and in HA? You are adding all values from the enum...
Can we not directly check if the model is part of the enum?

Reply:
to be precise we wouldn't need it in this PR, but in all follow up PRs platform PRs, which I can open in parallel if I have this in now.